atomic_reactor/tasks/binary.py

```diff
@@ -0,0 +1,103 @@
+"""
+Copyright (c) 2021 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from dataclasses import dataclass
+from typing import ClassVar
+
+from atomic_reactor import source
+from atomic_reactor.tasks import common
+from atomic_reactor.tasks import plugin_based
+
+
+@dataclass(frozen=True)
+class BinaryBuildTaskParams(common.TaskParams):
+    """Binary build task parameters (this task only uses common parameters)."""
+
+    # Validate with the source containers schema instead
+    user_params_schema: ClassVar[str] = "schemas/user_params.json"
+
+    @property
+    def source(self) -> source.Source:
+        """Return a dummy source for a binary container build."""
+        return source.DummySource(None, None, workdir=self.build_dir)
```

Comment:
No need to override, this can return a real Source just like the common TaskParams

Reply:
switched to common.TaskParams.